dashboard/app/views/shared/_extra_links.html.haml

```diff
@@ -0,0 +1,31 @@
+.extraLinks
```

Comment:
Need to also update the SCSS file? https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/assets/stylesheets/admin.css.scss